### PR TITLE
refactor: Use brackets instead of "Array"

### PR DESCRIPTION
**Pull request by ffflorian**
image

### CODE REVIEW

File: packages/core/src/main/client/ClientDatabaseRepository.ts [hunk 1]
```diff
@@ -55,7 +55,7 @@ class ClientDatabaseRepository {
   }
 
   public createClientList(userId: string, clientList: RegisteredClient[]): Promise<MetaClient[]> {
-    const createClientTasks: Promise<MetaClient>[] = [];
+    const createClientTasks: Array<Promise<MetaClient>> = [];
```
Comment (gregor) [inline packages/core/src/main/client/ClientDatabaseRepository.ts]: Why are you doing the reverse here then?

File: packages/cryptobox/src/main/store/CryptoboxCRUDStore.ts [hunk 2]
```diff
@@ -126,7 +126,7 @@ class CryptoboxCRUDStore implements ProteusSession.PreKeyStore {
    * Saves the serialised formats from a batch of PreKeys.
    */
   public save_prekeys(pre_keys: ProteusKeys.PreKey[]): Promise<ProteusKeys.PreKey[]> {
-    const promises: Promise<ProteusKeys.PreKey>[] = pre_keys.map(pre_key => this.save_prekey(pre_key));
+    const promises: Array<Promise<ProteusKeys.PreKey>> = pre_keys.map(pre_key => this.save_prekey(pre_key));
```
Comment (gregor) [inline packages/cryptobox/src/main/store/CryptoboxCRUDStore.ts]: And here...

File: packages/core/src/main/client/ClientDatabaseRepository.ts (hunk 1 above)
Comment (ffflorian) [reply packages/core/src/main/client/ClientDatabaseRepository.ts]: Because it's not a simple type.

Comment (bennyn) [reply packages/core/src/main/client/ClientDatabaseRepository.ts]: Is `Array<Promise<MetaClient>>` different from `Promise<MetaClient>[]`?

Comment (ffflorian) [reply packages/core/src/main/client/ClientDatabaseRepository.ts]: No, it's just a a different style of writing.

Comment (bennyn) [reply packages/core/src/main/client/ClientDatabaseRepository.ts]: Ok, then let's use `[]` for everything (no matter if complex type or simple type).

Comment (ffflorian) [reply packages/core/src/main/client/ClientDatabaseRepository.ts]: Okay!

Comment (ffflorian) [reply packages/core/src/main/client/ClientDatabaseRepository.ts]: Will you update the PR then?

Comment (bennyn) [reply packages/core/src/main/client/ClientDatabaseRepository.ts]: yes :)

**Pull request merged**